src/main/java/com/acme/dbo/txlog/ConsoleOutput.java

```diff
@@ -0,0 +1,7 @@
+package com.acme.dbo.txlog;
+
+public class ConsoleOutput {
+    public static <T> void sysOut(T message) {
```

Comment:
generic type T don't let message be primitive )

Reply:
yes, got it already, fixed)